Library/Homebrew/cli_parser.rb

```diff
@@ -4,6 +4,12 @@
 module Homebrew
   module CLI
     class Parser
+      class << self
+        def parse(&block)
```

Comment:
`def self.parse(&block)` would work, right?

Reply:
yeah, but I read that this way is more popular in RoR codebase :grimacing:

Reply:
Nevermind. Changing it to `def self.parse(&block)`